apps/mark-scan/frontend/src/pages/poll_worker_screen.tsx

```diff
@@ -417,14 +409,14 @@ export function PollWorkerScreen({
                 {getPollTransitionsFromState(pollsState).map(
                   (pollsTransition, index) => {
                     return (
-                      <React.Fragment
-                        key={`${pollsTransition}-directly-button`}
-                      >
-                        {index > 0 && ' or '}
-                        <UpdatePollsDirectlyButton
-                          pollsTransition={pollsTransition}
-                          updatePollsState={updatePollsState}
-                        />
+                      <React.Fragment key={`${pollsTransition}-button`}>
```

Comment:
Same comment here re the unneeded `React.Fragment`

Reply:
Cool, removing.